fasttrips/Assignment.py

```diff
@@ -43,7 +43,7 @@ class Assignment:
     CONFIGURATION_OUTPUT_FILE       = 'ft_output_config.txt'
 
     #: Configuration: Input network directory
-    INPUT_NETWORK_DIR               = None
+    INPUT_NETWORK_ARCHIVE               = None
```

Comment:
I realize that partridge currently takes this as a zip file, but that is not idea for a number of reasons (particularly for being able to see diffs on specific files) and I don't want to have to run a zip command before FT.  Therefore, I think we should either mod Partridge to be able to accept a directory + name-base (preferred) OR zip up required files within fast-trips.